bot.py

```
@@ -439,6 +439,297 @@ async def github_repo(interaction: discord.Interaction, repo: str):
+                description=f"**Branch:** `{branch}` **Path:** `/{subpath if subpath else ''}`",
+                color=EMBED_COLOR,
+                url=repo_url,
+            )
+
+            embed.set_author(
+                name=f"{owner}/{name} (branch: {branch})",
+                url=author_url,
+            )
+
+            embed.add_field(
+                name=f"Tree Structure (Max Depth: {max_depth}) – Part {idx}/{total_parts}",
```

Ruff (0.14.0)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)